frontend/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.1",
   "scripts": {
     "dev": "astro dev",
-    "build": "astro build && node dist/server/entry.mjs",
+    "build": "astro build",
```

Comment:
and you tried dynamic routes without this?

Reply:
yes